src/db/insert_data.py

```diff
@@ -1 +1,35 @@
-# Script to insert movies and interactions into MongoDB
\ No newline at end of file
+# Script to insert movies and interactions into MongoDB
```

Comment:
This might be best in the next PR....

Reply:
Though honestly there's like 83 thousand commits in this PR now, and it might not be worth it to try and remove this. Let's just try not to add any more features before we check this in.